pkg/rootless/rootless_linux.go

```diff
@@ -57,9 +57,15 @@ func IsRootless() bool {
 		rootlessGIDInit := int(C.rootless_gid())
 		if rootlessUIDInit != 0 {
 			// This happens if we joined the user+mount namespace as part of
-			os.Setenv("_CONTAINERS_USERNS_CONFIGURED", "done")
-			os.Setenv("_CONTAINERS_ROOTLESS_UID", fmt.Sprintf("%d", rootlessUIDInit))
-			os.Setenv("_CONTAINERS_ROOTLESS_GID", fmt.Sprintf("%d", rootlessGIDInit))
+			if err := os.Setenv("_CONTAINERS_USERNS_CONFIGURED", "done"); err != nil {
```

Comment:
This seems inconsistent with `runInUser()` where we return the `Setenv()` error.

Reply:
fair enough but there is no error return and I think people would be frustrated if we added one? what do others think?

Reply:
IMO, this doesn't matter much because this code should be going away quickly. We have 0.8.1 in the repos now; we don't need firewall code anymore.